rusk-abi/src/query/host.rs

```diff
@@ -71,3 +135,17 @@ pub fn verify_schnorr(msg: BlsScalar, pk: PublicKey, sig: Signature) -> bool {
 pub fn verify_bls(msg: Vec<u8>, apk: APK, sig: BlsSignature) -> bool {
     apk.verify(&sig, &msg).is_ok()
 }
+
+#[derive(Default)]
+struct DummyCircuit;
```

Comment:
not sure it is beneficial to introduce this construct

Reply:
Beneficial it is indeed absolutely not. This is done since we need to be able to specify which circuit type is being verified. It's not ideal for sure, but we can't get around it since `plonk` requires it. Take a look at [this issue](https://github.com/dusk-network/plonk/issues/713) if you'd like to contribute to have this solved.